src/common/IsolatedWorld.ts

```diff
@@ -164,20 +168,35 @@ export class IsolatedWorld {
 
   clearContext(): void {
     this.#document = undefined;
+    this.#puppeteerUtil = createDeferredPromise();
```

Comment:
should we call `dispose` on the previous handle? It looks like we should

Reply:
The handle is destroyed since the context is destroyed.